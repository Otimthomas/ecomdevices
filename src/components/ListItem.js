import React from "react";
import styled from "styled-components";

function ListItem({ name, image }) {
  return (
    <Wrapper>
      <div className="card">
        <img src={image} alt={name} />
        <p>{name}</p>
      </div>
    </Wrapper>
  );
}

export default ListItem;

const Wrapper = styled.div`
  .card {
    border: 1px solid #003931;
    border-radius: 10px;
    overflow: hidden;
    box-shadow: 2px 4px 6px #003931;
  }

  img {
    width: 250px;
    height: 250px;
    border-bottom: 2px solid #003931;
  }

  p {
    font-weight: 500;
    font-size: 25px;
    text-align: center;
    padding: 20px;
  }
`;
