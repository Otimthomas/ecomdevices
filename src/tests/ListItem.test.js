import ListItem from "../components/ListItem";
import renderer from "react-test-renderer";

describe("<ListItem />", () => {
  it("matches the snapshot", () => {
    let tree = renderer.create(<ListItem />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
