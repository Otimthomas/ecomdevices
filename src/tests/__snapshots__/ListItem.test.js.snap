// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListItem /> matches the snapshot 1`] = `
<div
  className="sc-bdvvtL efPRUK"
>
  <div
    className="card"
  >
    <img />
    <p />
  </div>
</div>
`;
